src/extension/extension.ts

```diff
@@ -88,6 +88,7 @@ export function activate(context: vscode.ExtensionContext) {
     // Buildifier formatting support
     vscode.languages.registerDocumentFormattingEditProvider(
       [
+        { language: "starlark"},
```

Comment:
Super-nit: Please add space before the closing brace (why is tslint not complaining?)

Reply:
User error on my end: I had my global tslint misconfigured so it wasn't using the right setup. Fixed.

Reply:
It should have complained on our CI too, though. I'll have to look into it.